src/store/flow/block-types/Core_SetContactPropertyStore.ts

```diff
@@ -4,11 +4,17 @@ import {IBlockConfig} from '@floip/flow-runner'
 import {cloneDeep} from 'lodash'
 import BaseStore, {actions as baseActions, IEmptyState} from '@/store/flow/block-types/BaseBlock'
 
+export interface IMultipleChoiceOption {
```

Comment:
I'd suggest we rename this with something like `IContactPropertyMultipleChoice` because the word `MultipleChoice` could be confusing as we have the internal vocabulary `MultipleChoiceQuestion`.
(with the prefix `I`, we can remove the last postfix `Option` I think)

Your thoughts?

- [ ] rename or reach out to me if you have a concern